node/src/accountant/payable_dao.rs

```diff
@@ -169,190 +202,192 @@ impl PayableDao for PayableDaoReal {
         .collect()
     }
 
-    fn top_records(&self, minimum_amount: u64, maximum_age: u64) -> Vec<PayableAccount> {
-        let min_amt = unsigned_to_signed(minimum_amount).unwrap_or(0x7FFF_FFFF_FFFF_FFFF);
-        let max_age = unsigned_to_signed(maximum_age).unwrap_or(0x7FFF_FFFF_FFFF_FFFF);
-        let min_timestamp = dao_utils::now_time_t() - max_age;
-        let mut stmt = self
-            .conn
-            .prepare(
-                r#"
-                select
-                    balance,
-                    last_paid_timestamp,
-                    wallet_address,
-                    pending_payable_rowid,
-                    pending_payable.transaction_hash
-                from
-                    payable
-                left join pending_payable on
-                    pending_payable.rowid = payable.pending_payable_rowid
-                where
-                    balance >= ? and
-                    last_paid_timestamp >= ?
-                order by
-                    balance desc,
-                    last_paid_timestamp desc
-            "#,
+    fn custom_query(&self, custom_query: CustomQuery<u64>) -> Option<Vec<PayableAccount>> {
+        let variant_top = TopStmConfig::new("last_paid_timestamp asc");
```

Comment:
Should this be `asc`? Or maybe `desc` with the newest payables first?

Reply:
I think it's legit. The user will rather come for looking up the accounts which have the oldest debts because those are which take more risks and so interest as well.